tests/unit_test/app/routes/test_git_tokens.py

```
@@ -0,0 +1,342 @@
+class TestGetByLabelEndpoint:
+	"""Test cases for GET /api/v1/git_tokens/{label} endpoint."""
+	
+	@staticmethod
+	def get_url(raw_label: str) -> str:
+		label = quote(raw_label, safe="")
+		return f"/api/v1/git_tokens/{label}"
+	
+	def test_get_git_label_by_label_success(self, client, mock_encryption_helper, mock_authenticated_user):
+		"""Test successful retrieval of git label by label."""
+		
+		mock_labels = [
```

Pylint (3.3.7)

[warning] Bad indentation. Found 2 spaces, expected 8

(W0311)